oak_launcher_utils/src/launcher/virtualized.rs

```diff
@@ -53,6 +53,11 @@ pub struct Params {
     /// Port to use for debugging with gdb
     #[arg(long = "gdb")]
     pub gdb: Option<u16>,
+
+    /// How much memory to give to the enclave binary, e.g., 256M (M stands for Megabyte, G for
+    /// Gigabyte).
+    #[arg(long)]
+    pub memory_size: Option<String>,
```

Comment:
If you don't provide an explicit `memory_size`, should we make it default to 1G? I think by default QEMU defaults to 256 MB or somesuch, which is definitely not enough.

Reply:
I think QEMU defaults to [128 MiB](https://github.com/project-oak/oak/issues/3786#issuecomment-1467698710), but currently that seems enough.

@tiziano88 had concerns about default allocating more memory in tests. I also think I would leave it for now until the tests fail. We can always follow up then and have everything in place.